contrib/kind.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Detect IP to use as API server
+API_IP=$(ip -4 addr | grep -oP '(?<=inet\s)\d+(\.\d+){3}' | grep -v "127.0.0.1" | head -n 1)
+if [ -z "$API_IP" ]; then
+  echo "Error detecting machine IP to use as API server"
+  exit 1
+fi
+
+sed -i "s/apiServerAddress.*/apiServerAddress: ${API_IP}/" kind.yaml
+
+# Create KIND cluster
+kind create cluster --name ovn --kubeconfig ${HOME}/admin.conf --image kindest/node:v1.16.4 --config=./kind.yaml
+export KUBECONFIG=${HOME}/admin.conf
+mkdir -p /tmp/kind
+sudo chmod 777 /tmp/kind
+echo $(kubectl get secrets -o jsonpath='{.items[].data.ca\.crt}') > /tmp/kind/ca.crt
+echo $(kubectl get secrets -o jsonpath='{.items[].data.token}') > /tmp/kind/token
```

Comment:
Just `kubectl ... > /tmp/...` should work? Don't need the `echo $()`

Reply:
right, will fix